src/java.base/share/classes/javax/crypto/KeyGenerator.java

```diff
@@ -293,8 +293,8 @@ public static final KeyGenerator getInstance(String algorithm,
      * Returns a {@code KeyGenerator} object that generates secret keys
      * for the specified algorithm.
      *
-     * <p> A new KeyGenerator object encapsulating the
-     * KeyGeneratorSpi implementation from the specified Provider
+     * <p> A new {@code KeyGenerator} object encapsulating the
+     * {@code KeyGeneratorSpi} implementation from the specified Provider
```

Comment:
Change "Provider" to start with lowercase or add `{@code }`

Reply:
Other methods seem to use "provider".

Reply:
Using lowercase.